site/docs/aws.md

```diff
@@ -444,7 +444,10 @@ spark-sql --packages org.apache.iceberg:iceberg-spark3-runtime:{{ versions.icebe
 [Hive](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-hive.html), [Flink](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-flink.html),
 [Trino](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-presto.html) that can run Iceberg.
 
-You can use a [bootstrap action](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-bootstrap.html) similar to the following to pre-install all necessary dependencies:
+Amazon EMR [added Apache Iceberg](https://aws.amazon.com/about-aws/whats-new/2022/01/amazon-emr-supports-apache-iceberg/) in distribution since 6.5.0.
+You can use Apache Spark 3.1.2 on EMR clusters with the Iceberg table format. Please refer the [official documentation](https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-iceberg-create-cluster.html) to create a cluster with Iceberg installed.
```

Comment:
Is the information about Apache Spark 3.1.2 included in the link? Or is there some sort of version compatibility matrix (or one that can be made for use eventually)?

The specification of Spark version 3.1.2 is going to likely leave readers with more questions re Spark 3.2 etc. That will be especially amplified in say, 6 months, when EMR will likely support Spark 3.2.x with some version of Iceberg (just a guess but seems reasonable).

That’s one of my few remaining “relative time” related concerns.

Also, thanks for updating the docs @rajarshisarkar! I certainly don’t think the docs won’t be updated for 3 years, but figured it might help as a way to think about it.

Reply:
Agreed, I have removed the Spark version.

Reply:
The official docs says that it's supported with  "Apache Spark 3": https://docs.aws.amazon.com/emr/latest/ReleaseGuide/emr-iceberg.html